src/Ops.Web/Files/CoverIssueBookmarklet.txt

```diff
@@ -14,6 +14,6 @@
 		}
 	}
 	else {
-		alert("Title is not displayed in the PAC.")
+		alert("This item is set to not Display in PAC in Polaris.")
```

Comment:
We show the error because, if they aren't marked as "Display in PAC" they're non-circulating items. There's no need to fix the cover image if customers can't see it or, check it out.

Reply:
@madison-boman @iafb Please enable the software to allow reporting of cover issues for items that have "Display in PAC" checked.

Reply:
@k7hpn Added this feature in commit 24c2fdfe5761ccc7f08e99b90f19d15627ab4604